docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,235 @@
+# Setting Up a Cluster
+You can setup a cluster in [Kubernetes](http://kubernetes.io/docs/whatisk8s/) or Jenkins.
+
+## Setting Up a Cluster on AWS using Kubernetes
+
+### Prerequisites
+- [kubectl](http://kubernetes.io/docs/user-guide/prereqs/)
+- an [AWS](http://aws.amazon.com) account
+- [AWS CLI](https://aws.amazon.com/cli/)
+
+### Create your cluster
+A cluster consists of a master API server and a set of worker VMs called nodes.
+
+#### Using get-kube
+To start up your cluster, run:
+
+```bash
+#Using wget
+export KUBERNETES_PROVIDER=aws; wget -q -O - https://get.k8s.io | bash
+#Using cURL
+export KUBERNETES_PROVIDER=aws; curl -sS https://get.k8s.io | bash
+```
+
+This process takes about 5 to 10 minutes. User credentials and security tokens are written in `~/.kube/config`.
+By default, the script will provision a new VPC and a 4 node k8s cluster in `us-west-2a` (Oregon) with EC2 instances running on Debian.
+
+NOTE: You can override the variables defined in `config-default.sh` to change this behavior. See the [Getting Started with Kubernetes guide](http://kubernetes.io/docs/getting-started-guides/aws/) for more details.
+
+
+
+### Setup secrets
+A Secret is an object that contains a small amount of sensitive data such as a password, a token, or a key. Such information can be put in a Pod specification or in an image, but putting it in a Secret object allows for more control over how it is used, and reduces the risk of accidental exposure. You can create a secret object in a file first, in json or yaml format, and then create that object.
+
+First, you will need to acquire your secrets.
+
+#### Kubernetes secrets
+Kubectl can be used to see your [Kubernetes secrets](http://kubernetes.io/docs/user-guide/secrets/walkthrough/).
+To get your k8s `host`, run:
+```bash
+$ kubectl cluster-info
+
+Kubernetes master is running at https://55.555.55.55
+```
+
+To get the k8s `<DEFAULT_TOKEN_NAME>`, run:
+```bash
+$ kubectl get secrets
+
+NAME                      TYPE                                  DATA      AGE
+datastore-dynamodb        Opaque                                2         19d
+db-user-pass              Opaque                                2         49d
+default-token-abc55       kubernetes.io/service-account-token   3         50d
```

Comment:
Is this the only token created by default? If so, should we only show this one, and not all the other secrets?